R/fit_baselines_one_location.R

```diff
@@ -62,6 +62,13 @@ fit_baselines_one_location <- function(model_variations,
                                        round_predictions = FALSE,
                                        seed = NULL) {
 
+  if (length(reference_date) > 1) {
+    cli::cli_abort("only one {.arg reference_date} may be provided")
+  } else {
+    reference_date <- reference_date |> # date to which horizons are relative
+      validate_ymd_date(arg_name = "reference_date")
```

Comment:
Very minor comment here -- it would seem that we could just call the function without using pipe, for brevity/clarity
```suggestion
    # date to which horizons are relative
    reference_date <- validate_ymd_date(reference_date, arg_name = "reference_date")
```